lms/static/js/dashboard/legacy.js

```diff
@@ -181,5 +210,84 @@
              $('#unenroll_course_number').text($(event.target).data('course-number'));
              $('#unenroll_course_name').text($(event.target).data('course-name'));
          });
+
+
+         function set_dialog_attributes(can_unenroll, is_paid_course, is_course_blocked, cert_name_long, course_number,
+                                        course_name, enrollment_mode, show_refund_option){
+
+             if (can_unenroll) {
+                 if (is_paid_course && show_refund_option) {
+                     if (!is_course_blocked) {
+                         return {
+                          'data-track-info': "Are you sure you want to unenroll from the purchased course "+ course_name + "s " + course_number + "s?",
```

Comment:
It looks like this message has been repeated multiple times. Can we refactor to aggregate?